boards/frdm-kw41z/include/board.h

```diff
@@ -86,6 +86,14 @@ extern "C"
 #define FRDM_NOR_SPI_CS                SPI_HWCS(0) /**< Flash CS pin */
 /** @} */
 
+/**
+ * @name    Accelerometer+magnetometer configuration
+ * @{
+ */
+#define FXOS8700_PARAM_ADDR         0x1f
```

Comment:
please put parentheses around this

Reply:
Is this really something that is worth blocking a PR for?
I will remove this change to avoid merge problems with master later on (#9442)

Reply:
nah sorry

Reply:
:+1: